ext/nnstreamer/extra/nnstreamer_protobuf.cc

```diff
@@ -99,16 +99,21 @@ gst_tensor_decoder_protobuf (const GstTensorsConfig *config,
 
   if (!gst_memory_map (out_mem, &out_info, GST_MAP_WRITE)) {
     nns_loge ("Cannot map output memory / tensordec-protobuf");
+    if (outbuf_size == 0)
+      gst_allocator_free (NULL, out_mem);
```

Comment:
Q) in this line, gst_allocator_free () and gst_memory_unref() is different?

Reply:
The expected role of both of them is the same. It's unified to `gst_memory_unref()`